dll/cpl/powercfg/powershemes.c

```diff
@@ -10,7 +10,7 @@
  */
 
 #include "powercfg.h"
-
+#include <debug.h>
```

Comment:
What's this for?

Reply:
Was for DPRINT1 that i initially pushed in the PR, then removed.... without removing debug.h sorry.
Do you want a PR for cleaning this or do we keep it ?